src/main/java/org/jenkinsci/plugins/scriptler/builder/ScriptlerBuilder.java

```diff
@@ -175,7 +180,7 @@ public Parameter[] getParameters() {
 
     @NonNull
     public List<Parameter> getParametersList() {
-        return Collections.unmodifiableList(parameters);
+        return parameters;
```

Comment:
Whether you should revert this or not would depend on whether or not you want callers to modify the list. Probably you do not, so this should be reverted.